### PR TITLE
rscape for structured alignments only

### PR DESCRIPTION
**Pull request by mmiladi**

### CODE REVIEW

**Comment by mmiladi** [comment]
This should avoid the crash, once rscape alignment has not structure.
I am not sure if the version number should be incremented or not..

File: tools/GraphClust/AlignCluster/call_rscape.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#! /bin/bash
```
Comment (bgruening) [inline tools/GraphClust/AlignCluster/call_rscape.sh]: Remove the space here.

File: tools/GraphClust/AlignCluster/test-data/alignment1.cons.tsv [hunk 2]
```diff
@@ -1,2 +1,2 @@
 	Background model	Columns	Combinations/Pair	Consensus MFE	Covariance contribution	Decision model	Energy contribution	G+C content	Mean pairwise identity	Mean single sequence MFE	Mean z-score	Prediction	Reading direction	SVM RNA-class probability	SVM decision value	Sequences	Shannon entropy	Structure conservation index	alignment	multihit per_species	num_human_seqs_all	num_seqs	num_seqs_all	rscape_#target_E-val	rscape_Found	rscape_MSA	rscape_PPV	rscape_SEN	rscape_TP	rscape_True	rscape_alen	rscape_avgid	rscape_method	rscape_nseq	rscape_out	cluster_bed	cluster_human_ids	cluster_human_loc
-0	dinucleotide	51	2.890	-7.640	-4.520	structural RNA alignment quality	-3.120	0.320	44.050	-5.080	0.220	RNA	forward	1.000	3.530	5.000	1.035	1.500	cluster.aln	0	0	5	12	0.050	5	result.aln_1	100.000	55.560	5	9	50	44.730	GTp	5	RESULTS/result.aln.sum
```
Comment (bgruening) [inline tools/GraphClust/AlignCluster/test-data/alignment1.cons.tsv]: Any idea why this has changed?

File: tools/GraphClust/AlignCluster/align_cluster.xml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<tool id="graphclust_align_cluster" name="Align GraphClust cluster " version="0.1" >
+<tool id="graphclust_align_cluster" name="Align GraphClust cluster " version="0.2" >
```
Comment (bgruening) [inline tools/GraphClust/AlignCluster/align_cluster.xml]: yeah, lets omit that bump

File: tools/GraphClust/AlignCluster/align_cluster.xml [hunk 4]
```diff
@@ -38,8 +38,8 @@
             #end if
         #end if
       "
-      && 
-      R-scape --outdir ./RESULTS/ result.aln.sth &&
+      &&
+      $__tool_directory__/call_rscape.sh ./RESULTS/ result.aln.sth &&
```
Comment (bgruening) [inline tools/GraphClust/AlignCluster/align_cluster.xml]: single quote ... the tool_directory could have spaces ...

File: tools/GraphClust/AlignCluster/test-data/alignment1.cons.tsv (hunk 2 above)
Comment (mmiladi) [reply tools/GraphClust/AlignCluster/test-data/alignment1.cons.tsv]: I think RNAz background model can slightly change, that's RNAz score and not Rscape. And also probability has not changed. So it should be fine.

**Pull request merged**